frontend/src/constants.js

```diff
@@ -1,4 +1,5 @@
 export const STATUS_CODE_CREATED = 201
 export const STATUS_CODE_CONFLICT = 409
+export const STATUS_CODE_NOTFOUND = 404

```

Comment:
Not sure if it's better to name it `STATUS_CODE_NOT_FOUND`. You may decide to change or not haha

Reply:
You're right, good catch!